Client/Frontend/Settings/CookieCutter/CookieCutterSettings.swift

```diff
@@ -17,10 +17,10 @@ struct CookieCutterSettings: View {
             Section(
                 footer:
                     Text(
-                        "When disabled, turns off Cookie Cutter on all sites. Changing this will reload your tabs."
+                        "When disabled, turns off Neeva Shield on all sites. Changing this will reload your tabs."
                     )
             ) {
-                Toggle("Cookie Cutter", isOn: $cookieCutterEnabled)
+                Toggle("Neeva Shield", isOn: $cookieCutterEnabled)
```

Comment:
```suggestion
                Toggle("Neeva Shield", isOn: $neevaShieldEnabled)
```

Maybe change this variable name.

Reply:
> Couple of comments:
> 
> * It looks like there are two different designs for the shield menu; when is one or the other used? Is it based on if the user is signed in?
> * "Neeva blocked cookie popup" sounds better as "Neeva blocked **a** cookie popup".

Thanks for the feedback! What shield menu do you refer to?

`Neeva blocked a cookie popup` sounds good, will change!